### PR TITLE
test: fix the testcase error on MariaDB.

### PR DESCRIPTION
**Pull request by ping-yee**
**Description**
See the first error of #7929 

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: tests/system/Database/Live/ForgeTest.php [hunk 1]
```diff
@@ -941,7 +941,7 @@ public function testAddFields(): void
                 ],
             ];
 
-            if (version_compare($this->db->getVersion(), '8.0.17', '>=')) {
+            if (version_compare($this->db->getVersion(), '8.0.17', '>=') && str_contains($this->db->getVersion(), 'MariaDB') === false) {
```
Comment (neznaika0) [inline tests/system/Database/Live/ForgeTest.php:944]: str_contains() not allowed for the current version of php 7.4

Comment (ping-yee) [reply tests/system/Database/Live/ForgeTest.php]: I have changed `strpos` function to handle this.

**Review by TimexPeachtree — APPROVED** [review]

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@ping-yee Thank you!

**Comment by neznaika0** [comment]
PR released, closed?

**Pull request merged**